emote/utils/model.py

```diff
@@ -0,0 +1,24 @@
+# This file contains codes/text mostly restructured from the following github repository
+# https://github.com/facebookresearch/mbrl-lib
+
+
+from typing import Union
+
+import numpy as np
+import torch
+
+
+def to_tensor(x: Union[torch.Tensor, np.ndarray]):
+    if isinstance(x, torch.Tensor):
+        return x
+    if isinstance(x, np.ndarray):
+        return torch.from_numpy(x)
+    raise ValueError("Input must be torch.Tensor or np.ndarray.")
+
+
+def to_numpy(x: Union[torch.Tensor, np.ndarray]):
+    if isinstance(x, torch.Tensor):
+        return x.detach().to("cpu").numpy()
```

Comment:
If we still need them, do we actually need to explicitly move to CPU before going to numpy?

Reply:
Yes, I think so. Here is an example:
b = a.detach().numpy()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: can't convert cuda:0 device type tensor to numpy. Use Tensor.cpu() to copy the tensor to host memory first.

Reply:
Ok, sounds good then!